app/src/common/shared/org/mozilla/vrbrowser/ui/NavigationBarWidget.java

```diff
@@ -386,21 +322,16 @@ private void exitResizeMode(boolean aCommitChanges) {
         }
         mIsResizing = false;
         mWidgetManager.finishWidgetResize(mBrowserWidget);
-        AnimationHelper.fadeIn(mFocusModeContainer, AnimationHelper.FADE_ANIMATION_DURATION, null);
+        AnimationHelper.fadeIn(mNavigationContainer, AnimationHelper.FADE_ANIMATION_DURATION, null);
         AnimationHelper.fadeOut(mResizeModeContainer, 0, null);
         mWidgetManager.popBackHandler(mResizeBackHandler);
     }
 
     private void setResizePreset(float aPreset) {
-        float worldWidth = WidgetPlacement.floatDimension(getContext(), R.dimen.browser_world_width);
-        float aspect = (float)SettingsStore.getInstance(getContext()).getWindowWidth() / (float)SettingsStore.getInstance(getContext()).getWindowHeight();
-        float worldHeight = worldWidth / aspect;
-        float area = worldWidth * worldHeight * aPreset;
-
-        float targetWidth = (float) Math.sqrt(area * aspect);
-        float targetHeight = (float) Math.sqrt(area / aspect);
-
-        mBrowserWidget.handleResizeEvent(targetWidth, targetHeight);
+        mBrowserWidget.setBrowserSize(
+                SettingsStore.getInstance(getContext()).getWindowWidth(),
+                SettingsStore.getInstance(getContext()).getWindowHeight(),
+                aPreset);
     }
 
     public boolean isInFocusMode() {
```

Comment:
should we rename this everywhere to `ResizeMode`?

Reply:
Yeah, not sure what's more clear, changed anyway.